src/containerapp/azext_containerapp/custom.py

```diff
@@ -5046,6 +5049,8 @@ def update_auth_config(cmd, resource_group_name, name, set_string=None, enabled=
     )
 
     containerapp_auth_decorator.construct_payload()
+    if containerapp_auth_decorator.get_argument_token_store() and containerapp_auth_decorator.get_argument_sas_url_secret() is not None:
```

Comment:
why not add it in construct_payload()?

Reply:
set_secrets func is defined in this file, to avoid dependency loop, so place the logic here

Reply:
`g.custom_command('set', 'set_secrets', exception_handler=ex_handler_factory())`
1.` set_secrets` is a function support command
2. `set_secrets` will send `ContainerAppClient.create_or_update`, I think if it set in construct_payload, maybe will excute twice in some senario. It should set in some place which not super() the GA function.